### PR TITLE
docs: use external references in documentation

### PR DESCRIPTION
**Pull request by manu0466**
## Description

Closes: #XXXX

This PR replace some information that are available in other repository with the appropriate references.
The replaced information are:
* Mainnet 
  * Genesis file 
  * Seed nodes
* Testnet
  * Genesis file
  * Seed nodes
  * State sync nodes
* Graphql **NOTE** This has been removed until we have the new DJuno documentation

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://docs.cosmos.network/v0.44/building-modules/intro.html)
- [ ] included the necessary unit and integration [tests](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by dadamu** [comment]
Both testnet and mainnet `State Sync` pages' example should be changed to be like:
```
enable = true

rpc_servers = "<rpc-address-1>:26657,<rpc-address-2>:26557"
trust_height = <block-height>
trust_hash = "<block-hash>"
trust_period = "336h0m0s"
```

As well as the peer example in the pages should be like:
```
persistent_peers = "<peer-node-id-1>@<peer-address-1>:26656,<peer-node-id-2>@<peer-node-address-2>:26656",....
```

**Review by RiccardoM — COMMENTED** [review]
Looks like the [testnet endpoint](https://6388b46e5b2ccc05e372160d--desmos-docs.netlify.app/testnet/endpoints) and [mainnet endpoint](https://6388b46e5b2ccc05e372160d--desmos-docs.netlify.app/mainnet/endpoints) are still there. We should probabily redicrect the user to the GitHub repo instead. The menu entries can be external links

File: docs/docs/05-testnet/05-endpoints.md [hunk 1]
```diff
@@ -3,26 +3,5 @@ id: endpoints
 title: Endpoints
 sidebar_label: Endpoints
 slug: endpoints
+href: https://github.com/desmos-labs/morpheus
```
Comment (RiccardoM) [inline docs/docs/05-testnet/05-endpoints.md:6]:
```suggestion
href: https://github.com/desmos-labs/morpheus/tree/main/morpheus-apollo-3#endpoints
```

File: docs/docs/06-mainnet/06-endpoints.md [hunk 2]
```diff
@@ -3,32 +3,5 @@ id: endpoints
 title: Endpoints
 sidebar_label: Endpoints
 slug: endpoints
+href: https://github.com/desmos-labs/mainnet/tree/main/references
```
Comment (RiccardoM) [inline docs/docs/06-mainnet/06-endpoints.md:6]:
```suggestion
href: https://github.com/desmos-labs/mainnet#endpoints
```

**Review by RiccardoM — COMMENTED** [review]
LGTM, great job with the endpoints handling. Just some changes to the URLs then it's ready to go

**Review by RiccardoM — APPROVED** [review]

**Pull request merged**